bedrock/base/management/commands/update_www_config.py

```diff
@@ -49,10 +50,22 @@ def output(self, msg):
             print(msg)
 
     def handle(self, *args, **options):
+        # Grab the existing monitor waitlist value so if something fails with the API, we don't lose it.
+        try:
+            monitor_waitlist_value = ConfigValue.objects.get(name=settings.MONITOR_SWITCH_WAITLIST).value
+        except ConfigValue.DoesNotExist:
+            monitor_waitlist_value = settings.MONITOR_SWITCH_WAITLIST_DEFAULT
+
         self.quiet = options["quiet"]
         repo = GitRepo(settings.WWW_CONFIG_PATH, settings.WWW_CONFIG_REPO, branch_name=settings.WWW_CONFIG_BRANCH, name="WWW Config")
         self.output("Updating git repo")
         repo.update()
+
+        # Check monitor API and set the waitlist switch as appropriate.
+        # Note: Doing this here since we want to run this regardless of whether or not there are changes in the config repo.
+        self.output("Checking monitor API...")
+        self.set_monitor_waitlist(monitor_waitlist_value)
```

Comment:
Because this is happening before the call to `refresh_db_values` below, this will be deleted if there are changes from the external repo. I like that it's happening first so it can be overriden via the repo, but we need to find a way for the value to persist. maybe passing in the determined value to `refresh_db_values` instead of setting it manually in `set_monitor_waitlist`?